src/helpers.test.js

```diff
@@ -6,95 +6,172 @@ import {
+            const foo = pick(bar, ['foo', 'foo2']);
 
-        expect(foo).toEqual({
-            foo: 'bar',
-            foo2: 'bar2',
+            expect(foo).toEqual({
+                foo: 'bar',
+                foo2: 'bar2',
+            });
         });
     });
+    describe('difference', () => {
+        it('should compere two array and return difference', () => {
+            const foo = ['foo', 'foo2', 'foo3', 'foo4'];
+            const bar = ['foo', 'foo2'];
 
-    it('should compere two array and return difference', () => {
-        const foo = ['foo', 'foo2', 'foo3', 'foo4'];
-        const bar = ['foo', 'foo2'];
-
-        expect(difference(foo, bar)).toEqual(['foo3', 'foo4']);
+            expect(difference(foo, bar)).toEqual(['foo3', 'foo4']);
+        });
     });
-
-    it('should wrap value to array', () => {
-        const foo = 'test';
-
-        expect(wrapToArray(foo, true)).toEqual(['test']);
-        expect(wrapToArray(foo)).toEqual('test');
+    describe('wrapToArray', () => {
+        it('should wrap value to array', () => {
+            const foo = 'test';
+            expect(wrapToArray(foo, true)).toEqual(['test']);
+        });
+        it('should not wrap value to array', () => {
+            const foo = 'test';
+            expect(wrapToArray(foo)).toEqual('test');
+        });
     });
+    describe('getDefaultValueForType', () => {
+        it('should get default value for type Number', () => {
+            const type = Number;
+            expect(Number.isNaN(getDefaultValueForType(type))).toEqual(true);
+        });
+        it('should get default value for type array of Numbers', () => {
+            const type = Number;
+            expect(Number.isNaN(getDefaultValueForType(type, true)[0])).toEqual(true);
+        });
 
-    it('should get default value for type', () => {
-        const number = Number;
-        const string = String;
-        const boolean = Boolean;
-        const date = Date;
-        const custom = { getDefaultValue: () => 'bar' };
+        it('should get default value for type String', () => {
+            const type = String;
+            expect(getDefaultValueForType(type)).toEqual('');
+        });
+        it('should get default value for type array of Strings', () => {
+            const type = String;
+            expect(getDefaultValueForType(type, true)).toEqual(['']);
+        });
 
-        expect(Number.isNaN(getDefaultValueForType(number))).toEqual(true);
-        expect(getDefaultValueForType(string)).toEqual('');
-        expect(getDefaultValueForType(boolean)).toEqual(false);
-        expect(getDefaultValueForType(custom)).toEqual('bar');
-        expect(getDefaultValueForType(date) instanceof Date).toEqual(true);
+        it('should get default value for type Boolean', () => {
+            const type = Boolean;
+            expect(getDefaultValueForType(type)).toEqual(false);
+        });
+        it('should get default value for type array of Strings', () => {
```

Comment:
`... for type array of Strings` --> `... for type array of Boolean`

Reply:
changed